ansible/group_vars/all.yml.sample

```diff
@@ -188,3 +188,21 @@ kafka_instances:
   key: kafka
   value: "true"
 kafka_node_label: 'kafka: "true"'
+
+################################################################################
+# CNF profile
+################################################################################
+# PTP interface (precision time protocol)
+ptp_interface: "eno1"
```

Comment:
Can you add some comment here as to what interface is expected to be listed here, if its the BM interface or something else?

Reply:
will do!